Cargo.toml

```diff
@@ -71,13 +72,13 @@ yubi = ["ethers-signers/yubi"]
 ## contracts
 abigen = ["ethers-contract/abigen"]
 
-
 [dependencies]
 ethers-contract = { version = "^0.5.0", default-features = false, path = "./ethers-contract" }
 ethers-core = { version = "^0.5.0", default-features = false, path = "./ethers-core", features = ["setup"] }
 ethers-providers = { version = "^0.5.0", default-features = false, path = "./ethers-providers" }
 ethers-signers = { version = "^0.5.0", default-features = false, path = "./ethers-signers" }
 ethers-middleware = { version = "^0.5.0", default-features = false, path = "./ethers-middleware" }
+ethers-derive-eip712 = { version = "0.1.0", path = "./ethers-core/ethers-derive-eip712"}
```

Comment:
no need here if we re-export the macro

Reply:
I don't think we can re-export the macro since it depends on ethers-core. From my understanding, proc-macros can only have one public export in the lib. We can remove the dependency here if we move the example into the `ethers-contracts/test` folder, which I think is preferable anyway.

Reply:
removed dependency here.